controllers/replicationdestination_controller.go

```diff
@@ -124,12 +122,19 @@ func (r *ReplicationDestinationReconciler) Reconcile(req ctrl.Request) (ctrl.Res
 				Message: err.Error(),
 			})
 	}
-
 	// Update instance status
 	statusErr := r.Client.Status().Update(ctx, inst)
 	if err == nil { // Don't mask previous error
 		err = statusErr
 	}
+	if !inst.Status.NextSyncTime.IsZero() {
```

Comment:
annoying question but for adding new clusters it would be nice to initially pull data upon replicationdestination object creation. The downside I see is if an organization updates at midnight every night on new clusters the data would not be available until then. Any thoughts @backube/scribemaintainers

Reply:
Agreed. I think the logic can be a bit more robust as well.

Reply:
Opened #98 to track